### ISSUE TITLE
EXT MARC conversion test issues with Ruby in Windows 10

### ISSUE THREAD

**Issue by takamarut** [LD4P/rdf2marc#43]
I installed Ruby 3.0 in Windows 10 and run bundle install for gems.
But, the EXT MARC conversions are unsuccessful.
I am not sure whether the command line is same as Linux system for Windows 10.
I tried the following command lines:
C:\rdf2marc\exe > rdf2marc https://api.stage.sinopia.io/....
C:\rdf2marc\lib > ruby rdf2marc.rb https://api.stage.sinopia.io/....
C:\rdf2marc\lib > ruby rdf2marc.rb
I got the following error messages:
![rdf2marcError](https://user-images.githubusercontent.com/50535652/106914357-f1279300-66d2-11eb-989d-b51be64c3f9a.PNG)
![rdf2marcError1](https://user-images.githubusercontent.com/50535652/106914369-f389ed00-66d2-11eb-8a95-8f77b9e43cc3.PNG)

Would it be possible to provide some instructions for Windows users?
Please advice me to run this EXT MARC conversion with Windows 10.

**Comment by takamarut** [comment]
I installed all gems which are listed in rdf2marc.rb.
Is this command line to execute "rdf2marc"? I got "loader" error. Would anyone advice me how to run rdf2marc? Thanks!
![image](https://user-images.githubusercontent.com/50535652/107168976-c4050a00-698a-11eb-8f0b-071bea336751.png)

**Comment by justinlittman** [comment]
I suspect that the rdf2marc executable will not work from the normal Windows commandline. I'm not at all familiar with Windows, but I understand it has a Linux subsystem. Can you try that?

**Comment by takamarut** [comment]
**Finally, rdf2marc works from the normal Windows command line!** 
I think Ruby gem, "byebug" installation with "bundle install" was the issue. After I installed Ruby gem, "byebug" with "gem install byebug" from the normal Windows command line, I can execute rdf2marc from the normal Windows command line. I did not have to install WLS (Windows Linux Subsystem).
I am using Ruby 3.0 from the Windows command line with this command, ruby exe/rdf2marc https://api.stage.sinopia.io/resource/042d6f65-16f3-451b-bacc-edccdc7cd553 This image is the output from the Windows command line. It seems that it has an issue with MeSH subject publication type heading. Also, vernacular characters don't show correctly in the output.
![rdf2marcJapaneseSerial](https://user-images.githubusercontent.com/50535652/111480934-dc461280-8708-11eb-88ab-e07774f82503.PNG)
